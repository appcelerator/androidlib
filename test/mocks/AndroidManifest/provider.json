{
	"application": {
		"provider": [
			{
				"android:authorities": "com.example.provider.cartoonprovider",
				"android:enabled": true,
				"android:exported": false,
				"android:grantUriPermissions": false,
				"android:icon": "@drawable/icon",
				"android:initOrder": 1,
				"android:label": "test provider 1",
				"android:multiprocess": false,
				"android:name": "testprovider1",
				"android:permission": "testPermission",
				"android:process": "testProcess",
				"android:readPermission": "somePermission",
				"android:syncable": false,
				"android:writePermission": "somePermission",
				"meta-data": [
					{
						"android:name": "zooVal",
						"android:value": "@string/kangaroo"
					},
					{
						"android:name": "zooRes",
						"android:resource": "@string/kangaroo"
					}
				],
				"path-permission": [
					{
						"android:path": "/test",
						"android:pathPattern": ".*",
						"android:pathPrefix": "ti_",
						"android:permission": "somePermission",
						"android:readPermission": "somePermission",
						"android:writePermission": "somePermission"
					},
					{
						"android:path": "/sample",
						"android:pathPattern": ".*",
						"android:pathPrefix": "test_",
						"android:permission": "somePermission",
						"android:readPermission": "somePermission",
						"android:writePermission": "somePermission"
					}
				]
			},
			{
				"android:authorities": "com.example.provider.cartoonprovider",
				"android:enabled": true,
				"android:exported": false,
				"android:grantUriPermissions": false,
				"android:icon": "@drawable/icon",
				"android:initOrder": 2,
				"android:label": "test provider 2",
				"android:multiprocess": false,
				"android:name": "testprovider2",
				"android:permission": "testPermission",
				"android:process": "testProcess",
				"android:readPermission": "somePermission",
				"android:syncable": false,
				"android:writePermission": "somePermission"
			}
		]
	}
}
