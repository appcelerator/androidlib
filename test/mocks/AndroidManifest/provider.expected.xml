<?xml version="1.0" encoding="utf-8"?>
<manifest>
  <application>
    <provider android:authorities="com.example.provider.cartoonprovider" android:enabled="true" android:exported="false" android:grantUriPermissions="false" android:icon="@drawable/icon" android:initOrder="1" android:label="test provider 1" android:multiprocess="false" android:name="testprovider1" android:permission="testPermission" android:process="testProcess" android:readPermission="somePermission" android:syncable="false" android:writePermission="somePermission">
      <meta-data android:name="zooVal" android:value="@string/kangaroo"/>
      <meta-data android:name="zooRes" android:resource="@string/kangaroo"/>
      <path-permission android:path="/test" android:pathPattern=".*" android:pathPrefix="ti_" android:permission="somePermission" android:readPermission="somePermission" android:writePermission="somePermission"/>
      <path-permission android:path="/sample" android:pathPattern=".*" android:pathPrefix="test_" android:permission="somePermission" android:readPermission="somePermission" android:writePermission="somePermission"/>
    </provider>
    <provider android:authorities="com.example.provider.cartoonprovider" android:enabled="true" android:exported="false" android:grantUriPermissions="false" android:icon="@drawable/icon" android:initOrder="2" android:label="test provider 2" android:multiprocess="false" android:name="testprovider2" android:permission="testPermission" android:process="testProcess" android:readPermission="somePermission" android:syncable="false" android:writePermission="somePermission"/>
  </application>
</manifest>
